### ISSUE TITLE
Access problems

### ISSUE THREAD

**Issue by DigitalBlade** [c0dycode/Borderlands-Hex-Multitool#41]
Hi c0dy!

You've heard it many times, but seriously, you are doing an incredible job, there's no way to measure just how much you improved many hours of my playtime. Anyway, I wanted to start a Borderlands 2 Reborn, but I could find the Hex tool. After few questionable decisions, I've uninstalled every trace of Hex tool, UCP and BLMM (maybe a bit overkill but my game uninstalled itself and besides I thought for some time that I didn't properly installed everything). After that I've downloaded Hex tool again and here I've encountered my problem. When I extract the files, I try to run the exe but I only get the message that my system couldn't get access to the exe file, that I may not have the privileges or something. I've tried a lot of things, uninstalling and reinstalling, running it as a administrator, opening it in different files but I always get that same message. So, Is there something very obvious that I'm missing? I've looked through a lot of videos and posts but really nothing helped, so I would be extremely thankful If you could. 

P.S. Sorry for my poor english...

**Comment by c0dycode** [comment]
Heya :)
I'm glad to hear that I could make the game(s) for enjoyable for you :)

Could it be possible that your antivirus is blocking it? That'd be my first guess.

**Comment by DigitalBlade** [comment]
Well, your first guess was correct! My antivirus, for some reason, quarantined the exe file and wouldn't allow for it to function properly. I have no idea why, when I had the multi tool and that antivirus earlier it worked perfectly, but oh well. I made the antivirus repaired itself, downloaded the Hex tool again and It's all working! I can't say how much I appreciate your fast response, I've been playing since I got it working and It's great. Again, thank you so much for your help, godspeed you magnificent bastard!
